src/components/Banner/Matomo.jsx

```diff
@@ -8,15 +8,16 @@ const Matomo = ({ matomoTracker }) => {
   const confirmTracking = !!Number(cookies.confirm_tracking);
   let tracker;
 
-  if (__CLIENT__) {
-    tracker = new matomoTracker.default({
-      urlBase: config.settings.DSGVOBanner.tracker.urlBase,
-      siteId: config.settings.DSGVOBanner.tracker.id,
-    });
-  }
-
   useEffect(() => {
-    if (confirmTracking) {
+    if (confirmTracking && __CLIENT__ ) {
```

Comment:
@fredvd useEffect hooks only run on the client (see the implementation of the useClient hook! https://github.com/plone/volto/blob/0987c96dbc4597b8ab36009af4e4bfa125f123ec/packages/volto/src/hooks/client/useClient.js#L5) so this is redundant.

```suggestion
    if (confirmTracking) {
```